docs/system-admin-guide/integrations/nextcloud/README.md

```diff
@@ -1,79 +1,102 @@
 
-![Nextcloud_connected_account](Nextcloud_connected_account.png)
+![module_activation](Settings_modules.png)
 
-The link will show here:
+![settings_files_storages](Settings_files_storages.png)
 
-![Navigation_link_OpenProject](Navigation_link_OpenProject.png)
+## How to reset your OAuth values
 
-By activating "enable unified search for tickets" in your personal settings, the Nextcloud dashboard will include OpenProject information in the the built-in universal search:
+If you need to reset the values of the Nextcloud integration (as an administrator), you can always reset the OAuth values from both sides of the integration by clicking on the **Reset OAuth values** (in Nextcloud) or **Replace OAuth values** (in OpenProject) buttons.
 
-![Unified_search](Unified_search.png)
+> **Important**: When you reset/replace these values, you will need to update the configuration with the new OAuth credentials from the side you are reseting. This will also require all users to re-authorize OpenProject to access their Nextcloud account by logging in again.
 
-**Set up of OAuth to OpenProject**
+![nextcloud_reset_OPOAuth](3_2_06 -NC_OP_OAuth_Replace.png)
 
-Within your Settings under Administration and then Connected Accounts you can set-up the OAuth authentication to your OpenProject instance.
+![nextcloud_reset_NCOAuth](3_2_07-NC__OAuth_Replace.png)
 
-![OAuth](OAuth.png)
+![openproject_reset_OAuth](3_4_03-OP_Replace_Alert.png)
 
-In OpenProject, add Nextcloud as application under Administration then Authentication and OAuth and enter the information in your Nextcloud instance.
 
-![OpenProject_OAuth](OpenProject_OAuth.png)
+## How to delete a file storage integration
 
-## Where do I find the Nextcloud integration in OpenProject?
+As an administrator, you can always delete the integration to Nextcloud using the **Delete** button in  the OpenProject settings or the **Reset** button in the Nextcloud settings.
 
-Further integration efforts are under way, which will deliver a Nextcloud integration also on the OpenProject side.
+> **Important:** If you perform this action, the integration will be reset and deleted and all settings and user connections that were created will be deleted. This means that should you want to reconfigure the integration, you will need complete the entire setup process once again.
 
-## What if project notifications are not displayed?
+## Using the integration
 
-If the notifications are not displayed in your Nextcloud dashboard, please check the following in your Nextcloud basic settings: in the background jobs, Cron must be activated.
+Once the file storage is added and enabled for projects, your users are able to take full advantage of the Nextcloud-OpenProject integration. For more information on how link Nextcloud files to work packages in OpenProject and access linked work packages in Nextcloud, please refer to the [Nextcloud integration user guide](../../../user-guide/nextcloud-integration/).
+.
```

Comment:
There's an unhappily single dot. :)

Reply:
The two things are already solved. Thanks @as-op